openfe/tests/storage/conftest.py

```diff
@@ -1 +1,102 @@
-from gufe.tests.conftest import *
+import pytest
```

Comment:
This is not the cleanest way to do this, but was the easiest way I could work out to get us out scope clashes 🤷🏽